app/views/api/ca_dashboard/stats/index.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+json.partial! 'api/v1/standard_response'
+json.stats do
+  json.totals @totals
```

Comment:
Just a quick question @aaronskiba , why do we have the value `"total_items": 0,` in the resulting json output? 

Is that something we need ?

Reply:
Good question; I guess that's just being inherited from `api/v1/standard_response`. I think it corresponds to the number of plans returned, which would always be zero for this new API endpoint:

I guess it does a similar thing when I check `api/v1/heartbeat`:
```
{
  "application": "DMP Assistant",
  "source": "GET /api/v1/heartbeat",
  "time": "2024-08-29T18:21:17+00:00",
  "caller": "142.244.108.150",
  "code": 200,
  "message": "OK",
  "total_items": 0,
  "items": [
  ]
}
```


Here's an API request for a specific plan:
```
{
  "application": "DMP Assistant",
  "source": "GET /api/v1/plans/id_redacted",
  "time": "2024-08-29T18:18:34+00:00",
  "caller": "DMP Administrator",
  "code": 200,
  "message": "OK",
  "page": 1,
  "per_page": 100,
  "total_items": 1,
  "items": [
    {
      "dmp": {
        "schema": "https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard/tree/master/examples/JSON/JSON-schema/1.0",
        "title": "v4.1.3 Test"
...
```